### PR TITLE
Migrate all orphaned resources to DefaultOrg

### PR DESCRIPTION
**Pull request by desa**
When users upgrade, all of their bolt resources will not belong to any
organization. This PR introduces a migration path where any orphaned,
resources without an organization, will become owned by the default
organization.

Connect #2198 
Connect #2311

### CODE REVIEW

File: bolt/client.go [hunk 1]
```diff
@@ -83,10 +83,19 @@ func (c *Client) Open(ctx context.Context) error {
 	}
 
 	// Runtime migrations
-	if err := c.DashboardsStore.Migrate(ctx); err != nil {
+	if err := c.OrganizationsStore.Migrate(ctx); err != nil {
 		return err
 	}
-	if err := c.OrganizationsStore.Migrate(ctx); err != nil {
+	if err := c.SourcesStore.Migrate(ctx); err != nil {
```
Comment (desa) [inline bolt/client.go]: SOURCES NEED DEFAULT ROLE

**Comment by jaredscheib** [comment]
Wait for clarification on outstanding questions in #2198.

**Comment by jaredscheib** [comment]
All newly authenticated users should be a SuperAdmin until the SuperAdmin changes the default Role for newly authenticated users, so this migration is fine. I'll review it shortly.

**Comment by desa** [comment]
> All newly authenticated users should be a SuperAdmin until the SuperAdmin changes the default Role for newly authenticated users, so this migration is fine. I'll review it shortly.

Should this be a part of this PR?

**Comment by jaredscheib** [comment]
Yeah, that would be appropriate @desa – good point.

**Comment by desa** [comment]
See https://github.com/influxdata/chronograf/issues/2311#issuecomment-343252759

- [ ] Add CLI option that specifies that everyone whoauthenticates is a superadmin. Default functionality is on.

File: server/server.go [hunk 2]
```diff
@@ -57,6 +57,8 @@ type Server struct {
 	CannedPath   string        `short:"c" long:"canned-path" description:"Path to directory of pre-canned application layouts (/usr/share/chronograf/canned)" env:"CANNED_PATH" default:"canned"`
 	TokenSecret  string        `short:"t" long:"token-secret" description:"Secret to sign tokens" env:"TOKEN_SECRET"`
 	AuthDuration time.Duration `long:"auth-duration" default:"720h" description:"Total duration of cookie life for authentication (in hours). 0 means authentication expires on browser close." env:"AUTH_DURATION"`
+	// TODO(desa): think of a better name
+	NewUsersNotSuperAdmin bool `long:"new-users-not-super-admin" description:"All new users will not be given the Super Admin privilege" env:"NEW_USRES_NOT_SUPER_ADMIN"`
```
Comment (jaredscheib) [inline server/server.go]: 1. typo at `NEW_USRES` --> `NEW_USERS`
1. make SUPER_ADMIN --> SUPERADMIN, thus: `NEW_USERS_NOT_SUPERADMIN`
1. make super-admin --> superadmin, thus: `new-users-not-superadmin`
1. replace the word `privilege` with `status`, per previous discussion

**Comment by desa** [comment]
I'm going to hold off on this one for the time being. I'm not a huge fan of this approach for the super admins. And this can reasonably be done at a later time. Unless anyone feels particularly strongly about this.

**Comment by jaredscheib** [comment]
What do you not like about this approach?

**Comment by jaredscheib** [comment]
Successfully migrated dashboards from non-multitenancy Chronograf to this version. They became owned by the Default org: the /dashboards response showed the `organization` field for the `Default` org with ID `0` with `viewer` as `role`.

LGTM

**Pull request merged**